frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateClusterCatalog/CreateControlPlane.test.tsx

```diff
@@ -49,17 +49,42 @@ describe('CreateControlPlane', () => {
     )
   }
 
-  test('can click hosted', async () => {
-    const initialNocks = [nockList(multiclusterEngine, mockMulticlusterEngine)]
-    render(<Component />)
-    await waitForNocks(initialNocks)
+  test('Hosted control plane card should be disabled when hypershift is disabled', async () => {
+    const { getByTestId } = render(<Component enableHypershift={false} />)
+    const card = getByTestId('hosted')
+    expect(isCardEnabled(card)).toBe(false)
+    expect(card).toHaveTextContent(
+      'Hosted control plane operator must be enabled in order to continueView documentation'
```

Comment:
This text needs to be improved (split sentences)

Reply:
Because View documentation is a link, for some reason in Jest it's processed this way, of course in the browser it shows differently
To avoid confusion I'm changing the assertion to just validate that the first part of the text appears, anyhow that's enough to make sure it works as expected